mapie/classification.py

```diff
@@ -633,11 +637,13 @@ def _fit_and_predict_oof_model(
         y_val = _safe_indexing(y, val_index)
 
         if sample_weight is None:
-            estimator = fit_estimator(estimator, X_train, y_train)
+            estimator = fit_estimator(
+                estimator, X_train, y_train, **fit_params
+                )
```

Comment:
Just a writing style suggestion:
```suggestion
            )
```